packages/atlas/src/components/_video/VideoPlayer/VideoPlayer.styles.ts

```diff
@@ -91,9 +93,51 @@ export const StyledSvgControlsSettingsSolid = styled(SvgControlsSettingsSolid)`
 export const StyledSvgControlsSettingsOutline = styled(SvgControlsSettingsOutline)`
   ${defaultIconColor};
 `
+export const StyledSvgControlsShare = styled(SvgControlsShare)`
+  ${defaultIconColor};
+`
 
 export const TRANSITION_DELAY = '50ms'
 
+export const EmbbeddedTopBar = styled.div<{ isFullScreen: boolean }>`
+  font-size: ${sizes(4)};
```

Comment:
Didn't we agree, we use `sizes` only for margins and paddings? Ignore this if I'm wrong